src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculable, Drawable {
```

Comment:
It's good that you're implementing interfaces to define behavior, but ensure that each interface has a single purpose. If `AreaCalculable` and `Drawable` have methods that are not conceptually close, they should not be implemented by the same class.